### PR TITLE
Jkantor/peer step zero

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Use the correct criterion option info when the point value of the median option is 0

JIRA: [AU-577](https://2u-internal.atlassian.net/browse/AU-577)

**Testing Instructions**

(This one's a pain, I don't expect anyone to manually test it. But if you so desire:)

- Make a peer ORA with a rubric with a criterion option for zero points.
- Have a submission be peer graded and recieve a final grade for which one or more criteria recieve the zero point option
- A small text bug can be seen in the "final grade" section and in the "manage individual learner" section (see ticket)
- check out this branch
- The bug should be resolved.

Screen Shot 2022-07-20 at 9 58 10 AM

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by nsprenkle — APPROVED** [review]
LGTM with the obvious "do a version bump"
